src/vma/sock/sockinfo.cpp

```diff
@@ -967,6 +975,8 @@ bool sockinfo::attach_as_uc_receiver(role_t role, bool skip_rules /* = false */)
 			}
 		}
 	}
+	si_logdbg("Attached to specific local if: (%d.%d.%d.%d) addr: %s", NIPQUAD(local_if), addr.to_str());
```

Comment:
The log messages are better without the change.
While _local_if_ == INADDR_ANY, we will call _attach_receiver()_ for each offloaded ip but we will only print the information of the last ip
what do you think ?